template_handler/cdk.py

```diff
@@ -1,319 +0,0 @@
-import aws_cdk
```

Comment:
I renamed the root directory on this file because the import path for users to be able to import the stack class defined in it was `from template_handler import ...` which didn't really make a lot of sense. Now users can `from sceptre_cdk_handler import SceptreCdkStack` which is a much better API.